ui/settingsdlg.ui

```diff
@@ -1556,6 +1556,13 @@ The qube must be running to disable seamless mode; this setting is not persisten
    </item>
   </layout>
  </widget>
+ <customwidgets>
+  <customwidget>
+   <class>SizeSpinBox</class>
+   <extends>QSpinBox</extends>
+   <header>qubesmanager/utils.h</header>
```

Comment:
Didn't you forgot to add this file? Or is it re-interpreted as utils.py?

Reply:
Nope, it is interpreted as utils.py by PyQt itself. Magic, basically :)